test/exercism/homework_test.rb

```diff
@@ -0,0 +1,69 @@
+require_relative '../test_helper'
+require_relative '../../lib/exercism'
+require_relative '../../lib/exercism/homework'
+
+class HomeworkTest < Minitest::Test
+  def alice
+    @alice ||= User.create(username: 'alice')
+  end
+
+  def teardown
+    super
+    @alice = nil
+    @language = nil
+    @skipped = nil
+    @fetched = nil
+    @recent = nil
+  end
+
+  def create_exercises
+    @language = 'ruby'
+    time = Time.now
+
+    @skipped = UserExercise.create(user: alice,
+                                   language: @language,
+                                   slug: 'leap',
+                                   skipped_at: time)
+
+    @fetched = UserExercise.create(user: alice,
+                                   language: @language,
+                                   slug: 'clock',
+                                   fetched_at: time)
+
+    @recent = UserExercise.create(user: alice,
+                                  language: @language,
+                                  slug: 'gigasecond',
+                                  last_iteration_at: time)
+  end
+
+  def test_status_with_progress_on_track
+    create_exercises
```

Comment:
I'm assuming with MiniTest any objects created in the tests are destroyed after the test has finished running, like with Rspec, right? I would have expected failures in this test if that weren't the case, and I didn't get any, so for now I'm assuming this is a valid way to go with this.